### PR TITLE
Statin Measure

### PR DESCRIPTION
**Pull request by utsavpaudel**
## Describe your changes
Please include a summary of any changes.

## How has this been tested?
Please describe the tests you ran to verify your changes.  Provide instructions or code to reproduce output.

## Reviewer focus
Please summarize the specific items you’d like the reviewer(s) to look into.

## Checklist before requesting a review
- [ ] I have added at least one Github label to this PR (bug, enhancement, breaking change,...)
- [x] My code follows [style guidelines](https://thetuvaproject.com/contribution-guides/development-style-guide)
- [x] (New models) [YAML files](https://github.com/tuva-health/tuva/blob/main/models/hcc_suspecting/hcc_suspecting_models.yml) are categorized by sub folder and models listed in alphabetical order
- [x] (New models) I have added a [config](https://github.com/tuva-health/tuva/blob/main/models/hcc_suspecting/final/hcc_suspecting__list.sql) to each new model to enable it for claims and/or clinical data
- [x] (New models) I have added the variable `tuva_last_run` to the final output
- [ ] (Optional) I have recorded a Loom to explain this PR

### Package release checklist
- [ ] I have updated [dbt docs](https://www.notion.so/tuvahealth/Building-dbt-Docs-16df2f00df244f29b9d6756d8adbc2d9)
- [ ] I have updated the version number in the `dbt_project.yml`

### CODE REVIEW

File: models/quality_measures/final/quality_measures__summary_wide.sql [hunk 1]
```diff
@@ -127,5 +141,6 @@ select
     , cqm_236
     , nqf_0053
     , cbe_0055
+    , cqm_438
     , '{{ var('tuva_last_run')}}' as tuva_last_run
 from add_data_types
```
Comment (deepsonshrestha) [inline models/quality_measures/final/quality_measures__summary_wide.sql]: add EOL

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+        , performance_period_begin
+        , performance_period_end
+        , measure_id
+        , measure_name
+        , measure_version
+        , 3 as criteria
+    from {{ ref('quality_measures__int_cqm438_denominator_criteria3') }}
+
+)
+
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql:47]: remove this empty line

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+     )
+
+)
+
+, procedure_encounters as (
+
+    select 
+          patient_id
+        , procedure_date as min_date
+        , procedure_date as max_date
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql:93]: use proper aliasing

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+    from {{ref('quality_measures__stg_core__procedure')}} proc
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql]: change alias to be in human readable format

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator_criteria2.sql [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+        , procedures.procedure_date as evidence_date
+    from procedures
+         inner join cholesterol_codes
+             on procedures.code = cholesterol_codes.code
+             and procedures.code_type = cholesterol_codes.code_system
+
+)
+
+, labs as (
+    select
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator_criteria2.sql:82]: add empty line here

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator_criteria2.sql [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+        patient_id
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator_criteria2.sql]: format this with one tab idention

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator_criteria2.sql [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+)
+
+, cholesterol_labs as (
+
+    select 
+          patient_id
+        , evidence_date
+    from cholesterol_tests_with_result
+    where rn= 1 
+        and evidence_value > 190
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator_criteria2.sql:122]: I think `evidence_value` needs to be type-casted into integer before comparing with the numerical

Comment (deepsonshrestha) [reply models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator_criteria2.sql:122]: Or could we ensure this column is entirely numeric?

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_long.sql [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+
+    select distinct patient_id
+    from {{ ref('quality_measures__stg_core__patient') }}
+
+)
+
+, denominator as (
+
+    select
+          *
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_long.sql:17]: specify the columns referenced

File: models/quality_measures/quality_measures_models.yml
Comment (deepsonshrestha) [inline models/quality_measures/quality_measures_models.yml:1]: order the created models in ascending order

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator_criteria2.sql (hunk 7 above)
Comment (utsavpaudel) [reply models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator_criteria2.sql:122]: The regex ensures we have numeric type in the evidence value. This has been extensively tested in our environment.

**Pull request merged**